apps/common/src/python/mediawords/util/config/common.py

```diff
@@ -155,6 +155,14 @@ def password() -> str:
         """Password."""
         return ''
 
+    @staticmethod
+    def unsubscribe_address() -> str:
```

Comment:
Here despite the method being called `unsubscribe_address()` (if I read that I would expect it to return just the `support@example.com` email address) you're returning the full `mailto:` link including the proposed subject line.

I would make the `unsubscribe_address()` method return *just* the address that the unsubscribe emails should be sent to, and then let the caller (`send_email()`) decide where to put this address and what subject does it want to append to it (if any) as this is what a method (or rather the name of it) promises.